coffee-dto/coffee-dto-xsd/src/main/resources/xsd/hu/icellmobilsoft/coffee/dto/common/common.xsd

```diff
@@ -296,6 +296,7 @@
 	<xsd:simpleType name="PageType">
 		<xsd:restriction base="xsd:int">
 			<xsd:minInclusive value="1" />
+            <xsd:maxInclusive value="21474836" />
```

Comment:
Could you add a few more spaces to the beginning of the line to make it the same as one above?